libraries/ssh_crypto.rb

```diff
@@ -197,6 +198,8 @@ def valid_privseparation
       when /7\./
         ps = ps59
       end
+    when 'alpine'
```

Comment:
I think we can remove the alpine section here

Reply:
Depending on the comment above, we need someway to make sure `nil` is returned so that OpenSSH >= 7.5 results in the check `its('UsePrivilegeSeparation') { should eq(nil) }`